src/storage.js

```diff
@@ -244,6 +248,9 @@ export class LocalStorageStore {
    */
   async handleMigrationOldRegisteredTokens(storage) {
     const oldTokens = this.getItem('wallet:tokens');
+    if (!oldTokens) {
+      return;
+    }
     for (const token of oldTokens) {
       await storage.registerToken(token);
     }
```

Comment:
When migrating a previous wallet that only had HTR or was empty this would fail since we cannot create a for..of loop for null or undefined